packages/sources/tiingo-state/src/transport/utils.ts

```diff
@@ -0,0 +1,18 @@
+export const wsMessageContent = (
+  eventName: 'subscribe' | 'unsubscribe',
+  apiKey: string,
+  thresholdLevel: number,
+  base: string,
+  quote: string,
+  skipSlash = false,
+) => {
+  const ticker = skipSlash ? `${base}${quote}` : `${base}/${quote}`
+  return {
+    eventName,
+    authorization: apiKey,
+    eventData: {
+      thresholdLevel,
```

Comment:
```suggestion
      thresholdLevel: thresholdLevel,
```
(unless I'm misunderstanding)

Reply:
in JS/TS it'll populate the json with named params thresholdLevel [key]: [value]